### PR TITLE
Vendors in get_limb_coordinates to produce plot_limb method

### LINKED ISSUE #60: Default colour of Spline is based on value
In Pyvista, the default value for color that Spline takes is based on it's arc length values (created when a spline is made from the existing points).
Both quadrangle, limb and maybe grid if we implement it would be using Spline, should we settle on a default colour instead of this?
image

### PR DESCRIPTION
**Pull request by jeffreypaul15**
Created a separated utils.py to hold `get_limb_coordinates()` for now.
Also fixes #60 by defaulting the colour to white instead.
- [x] Tests
- [x] Example
- [x] Figure test

@dstansby Passing a map to the method should be okay I'm assuming?

### CODE REVIEW

File: sunkit_pyvista/utils.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+__all__ = ['get_limb_coordinates']
+
+
+@u.quantity_input
+def get_limb_coordinates(observer, rsun=constants.radius, resolution=1000):
+
+    """
+    Get coordinates for the solar limb as viewed by a specified observer.
+    Parameters
```
Comment (nabobalis) [inline sunkit_pyvista/utils.py:16]:
```suggestion

    Parameters
```

File: sunkit_pyvista/tests/test_pyvista.py [hunk 2]
```diff
@@ -178,7 +178,12 @@ def test_loop_through_meshes(plotter):
...
     inner_block = pv.MultiBlock([sphere])
     outer_block = pv.MultiBlock([inner_block, sphere2])
-
     plotter._loop_through_meshes(outer_block)
 
     assert plotter.plotter.mesh.center == [0, 1, 1]
+
+
+def test_plot_limb(aia171_test_map, plotter):
+    plotter.plot_map(aia171_test_map)
```
Comment (nabobalis) [inline sunkit_pyvista/tests/test_pyvista.py:187]: I assume this test will be changed in future?

Comment (jeffreypaul15) [reply sunkit_pyvista/tests/test_pyvista.py:187]: Why though?

Comment (nabobalis) [reply sunkit_pyvista/tests/test_pyvista.py:187]: How does this test plot_limb?

Comment (jeffreypaul15) [reply sunkit_pyvista/tests/test_pyvista.py:187]: Oh, my apologies, I thought I typed `plot_limb`. Completely missed this one.

File: sunkit_pyvista/plotter.py [hunk 3]
```diff
@@ -377,3 +383,27 @@ def load(self, filepath):
...
+        ----------
+        m : `sunpy.map.Map`
+            Map's limb to be plotted.
+        radius : `float`
+            Radius of the `pyvista.Spline` used to create the limb.
+            Defaults to ``0.02`` times the radius of the sun.
+        **kwargs : Keyword arguments are handed to `pyvista.Plotter.add_mesh`.
+        """
+        limb_coordinates = get_limb_coordinates(m.observer_coordinate, m.rsun_meters,
+                                                resolution=1000)
```
Comment (dstansby) [inline sunkit_pyvista/plotter.py:401]: We should allow the user to specify the resolution as a keyword argument to this method.

**Review by dstansby — CHANGES_REQUESTED** [review]
👍 nice work, just one request for a change

File: sunkit_pyvista/plotter.py (hunk 3 above)
Comment (jeffreypaul15) [reply sunkit_pyvista/plotter.py:401]: Ah, we're using Spline here so I don't think resolution matters right?

Comment (dstansby) [reply sunkit_pyvista/plotter.py:401]: I guess that's true

**Comment by dstansby** [comment]
Sorry, this is going to need a rebase, and the test figure will need generating again as I've just merged in my polydata PR

**Comment by jeffreypaul15** [comment]
> Sorry, this is going to need a rebase, and the test figure will need generating again as I've just merged in my polydata PR

No worries

**Review by dstansby — APPROVED** [review]

**Pull request merged**